docs/pages/deploy-a-cluster/aws-gslb-proxy-peering-ha-deployment.mdx

```diff
@@ -0,0 +1,252 @@
+---
+title: "AWS Multi-Region High Availability Deployment Guide"
+description: "Deploying a Proxy-peered High Availability Teleport Cluster using Route 53 to create Global Server Load Balancing"
+---
+
+## Overview
+
+This deployment architecture features two important design decisions:
+
+1. AWS Route 53 Latency based routing is used for Global Server Load Balancing 
+   [GSLB](https://www.cloudflare.com/learning/cdn/glossary/global-server-load-balancing-gslb/). 
```

Comment:
Here we mention AWS and then link to a Cloudflare page. Is there an AWS GSLB doc we could link to instead?

Reply:
@zmb3 AWS doesn't have documentation on it. My understanding is that GSLB is an architecture type that is created through the use of DNS geographic or latency routing. It isn't a service or product actually offered by AWS.